src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System/Runtime/InteropServices/JavaScript/JavaScriptTestHelper.mjs

```diff
@@ -246,6 +246,22 @@ export function invoke2(arg1, name) {
     return res;
 }
 
+export function returnResolvedPromise() {
+    return Promise.resolve();
+}
+
+export async function invokeReturnCompletedTask() {
+    const JavaScriptTestHelper = dllExports.System.Runtime.InteropServices.JavaScript.Tests.JavaScriptTestHelper;
+    const p = JavaScriptTestHelper['ReturnCompletedTask'];
+    Promise.race([p, delay(100)])
```

Comment:
`await Promise.race`

Reply:
Right, it should be there. I don't see any change in behavior, though

Reply:
it's `()`! I save function as `p` but never execute it. Sorry, the commit is flying in a sec